### ISSUE TITLE
Uncaught TypeError: Cannot read property 'prototype' of undefined

### ISSUE THREAD

**Issue by jorgechavz** [cdnjs/atom-extension#8]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.210.0
**System**: Mac OS X 10.10.2
**Thrown From**: [cdnjs](https://github.com/cdnjs/atom-extension) package, v1.4.0

### Stack Trace

Uncaught TypeError: Cannot read property 'prototype' of undefined

```
At /Users/Jorge/.atom/packages/cdnjs/lib/cdnjs-view.coffee:4

TypeError: Cannot read property 'prototype' of undefined
  at __extends (/Users/Jorge/.atom/packages/cdnjs/lib/cdnjs-view.coffee:4:199)
  at /Users/Jorge/.atom/packages/cdnjs/lib/cdnjs-view.coffee:11:3
  at Object.<anonymous> (/Users/Jorge/.atom/packages/cdnjs/lib/cdnjs-view.coffee:201:5)
  at Object.<anonymous> (/Users/Jorge/.atom/packages/cdnjs/lib/cdnjs-view.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/Jorge/.atom/packages/cdnjs/lib/cdnjs.coffee:1:13)
  at Object.<anonymous> (/Users/Jorge/.atom/packages/cdnjs/lib/cdnjs.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:709:34)
  at Package.module.exports.Package.activateConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:257:12)
  at PackageDetailView.module.exports.PackageDetailView.activate (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:163:26)
  at PackageDetailView.module.exports.PackageDetailView.updateInstalledState (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:201:12)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-detail-view.js:235:24
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/emitter.js:155:30
  at Array.forEach (native)
  at PackageManager.module.exports.Emitter.emitSlow (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/emitter.js:154:47)
  at PackageManager.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/emitter.js:107:30)
  at PackageManager.module.exports.PackageManager.emitPackageEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:546:19)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:369:26
  at exit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:53:16)
  at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:213:47)
  at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:235:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:169:7)
  at Process.ChildProcess._handle.onexit (child_process.js:1057:12)

```

### Commands

```
     -2:13 snippets:expand (atom-text-editor.editor.is-focused)
     -2:13 editor:indent (atom-text-editor.editor.is-focused)
     -2:12.7.0 core:save (atom-text-editor.editor.is-focused)
     -1:52.5.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -1:52.5.0 editor:newline (atom-text-editor.editor.is-focused)
     -1:50.3.0 emmet:expand-abbreviation-with-tab (atom-text-editor.editor.is-focused)
     -1:49.4.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -1:49.2.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -1:39.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  3x -0:45.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:42.8.0 emmet:expand-abbreviation-with-tab (atom-text-editor.editor.is-focused)
     -0:42.2.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:41.9.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:41.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:40.8.0 core:save (atom-text-editor.editor.is-focused)
     -0:03.6.0 deprecation-cop:view (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-atom-dark-ui)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ],
    "ignoredNames": [
      ".DS_Store"
    ]
  }
}
```

### Installed Packages

```coffee
# User
Stylus, v1.0.0
atom-jade, v0.3.0
color-picker, v2.0.4
css-snippets, v0.8.0
emmet, v2.3.10
language-bats, v0.1.0
language-elixir, v0.5.1
language-stylus, v0.2.0

# Dev
No dev packages
```

**Comment by ksprwhite** [comment]
I have same issue, with messages about deprecated calls.

atom v0.210.0
ubuntu-gnome 64 bits

**Comment by thomasdavis** [comment]
Damn, I'm pretty low on time at the moment. Can anyone pinpoint the problem?

**Comment by ksprwhite** [comment]
solved to me after upgrade api to 1.0

**Comment by thomasdavis** [comment]
@ksprwhite solved this in a pull request. I've published 1.5.0 and should be available via apm now. Thanks gents!
